lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -113,7 +113,7 @@ class DetailsRenderer {
     }
 
     const element = this._dom.createElement('div', 'lh-text__url');
-    element.appendChild(this._renderText(displayedPath));
+    element.appendChild(this._renderAnchor(displayedPath, url));
```

Comment:
seems like we should rename this method now ;)

maybe we can also share some logic with `renderLink` at this point since I was about to make `_renderAnchor` comments about falling back to text on invalid, etc

Reply:
ah, didn't see that method. done.